scripts/asr_language_modeling/ngram_lm/install_beamsearch_decoders.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
-# install Boost package
-sudo apt-get update
-sudo apt-get install swig build-essential libboost-all-dev cmake zlib1g-dev libbz2-dev liblzma-dev
+cd /workspace/nemo
+apt-get update && apt-get upgrade -y && apt-get install -y liblzma-dev && rm -rf /var/lib/apt/lists/*
+
```

Comment:
Please add some docs in the top of the file on what it does and how they can use it.

Reply:
added